frontend/src/pages/instance/Snapshots/dialogs/SnapshotCreateDialog.vue

```diff
@@ -10,7 +10,7 @@
                     </template>
                 </FormRow>
                 <FormRow v-model="input.setAsTarget" type="checkbox" data-form="snapshot-name">
-                    <span v-ff-tooltip:right="'If checked, all devices in this team will be restarted on this snapshot.'" class="">
+                    <span v-ff-tooltip:right="'If checked, all devices in this instance will be restarted on this snapshot.'" class="">
```

Comment:
Well spotted - the original comment was not current as it never applied to all devices in the team. Suggest a slightly tweak:
```suggestion
                    <span v-ff-tooltip:right="'If checked, all devices assigned to this instance will be restarted on this snapshot.'" class="">
```